Lib9c/Action/DailyReward.cs

```diff
@@ -65,13 +66,15 @@ public override IAccountStateDelta Execute(IActionContext context)
             avatarState.dailyRewardReceivedIndex = context.BlockIndex;
             avatarState.actionPoint = gameConfigState.ActionPointMax;
 
+            var runeAmount = gameConfigState.DailyRuneRewardAmount;
```

Comment:
it can be 0 or null when before patch `GameConfigSheet`. maybe raise Exception when this case. could you add this test case?